lib/rules/no-unescaped-entities.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @fileoverview HTML special characters should be escaped.
+ * @author Patrick Hayes
+ */
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+var DEFAULTS = ['>', '<', '"', '\'', '}'];
```

Comment:
Yeah, I also found the omission of `{` curious. In any case, it would be really great if the documentation included an example of why each of the defaults can be problematic.